ibis/mapd/tests/test_client.py

```diff
@@ -133,3 +135,21 @@ def test_create_table_schema(con):
         assert isinstance(t.w, ir.MultiPolygonColumn)
     finally:
         con.drop_table(t_name)
+
+
+@pytest.mark.parametrize(
+    'sql',
+    [
+        'select * from functional_alltypes limit 10--test',
+        'select * from functional_alltypes \nlimit 10\n;',
+        'select * from functional_alltypes \nlimit 10;',
+        'select * from functional_alltypes \nlimit 10;--test',
+    ],
+)
+@pytest.mark.skipif(
+    sys.version_info < (3, 6),
+    reason='`sql` method just available for Python 3.6 or greater.',
+)
+def test_sql(con, sql):
```

Comment:
Do these tests actually fail without the regex munging?

Reply:
yes ... also for another backend if I recall correctly .. that is why I needed to move these tests from  `all` to `mapd`

Reply:
https://dev.azure.com/ibis-project/ibis/_build/results?buildId=1286&view=logs

here you can see that these tests failed for impala also (LinuxTest py37)